bin/countperl

```diff
@@ -7,6 +7,7 @@ use Carp qw();
 use Readonly;
 use Pod::Usage qw(pod2usage);
 use Perl::Metrics::Simple qw(0.13);
+use JSON::PP qw(encode_json);
```

Comment:
I'll need to add this to Build.PL and Makefile.PL
(I also need to update the `Changes` file)